<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntranceWindow</class>
 <widget class="QWidget" name="EntranceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>892</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>EntranceWindow</string>
  </property>
  <widget class="QToolButton" name="MusicButton_">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/musicSoundTrack.png</normaloff>:/img/resource/musicSoundTrack.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>57</width>
     <height>57</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="ShopButton_">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Shop_enterScene.png</normaloff>:/img/resource/Shop_enterScene.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="HelpButton_">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Help_enterScene.png</normaloff>:/img/resource/Help_enterScene.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>52</width>
     <height>52</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="SaveButton_">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/Save_eS.png</normaloff>:/img/resource/Save_eS.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="StartButton_">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>630</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/resource/NewGame.png</normaloff>:/img/resource/NewGame.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>130</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
